web-ng/README.md

```diff
@@ -0,0 +1,39 @@
+# Dev Environment Setup
+
+Install ng CLI using npm:
+```
+npm install -g @angular/cli
+```
+
+Upgrade Node.js to the newest version using nvm: (Angular requires Node.js version 10.9.0 or later.)
+```
+nvm install node --reinstall-packages-from=node
+```
+
+# WebNg
+
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 8.3.14.
+
+## Development server
+
+Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
+
+## Code scaffolding
+
+Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
```

Comment:
Maybe mention there are generator plugins for popular text editors?

Reply:
I am not quite sure what you mean here. Can you elaborate more?

Reply:
I was suggesting to mention that they could use editor plugins to do the same, but I think we can leave that as an exercise for the reader.